tests/test_cli/test_remove/test_dependencies.py

```diff
@@ -167,7 +167,9 @@ def test_removed_with_dependencies(self):
 
         Done with mocking _add_item_deps to avoid dependencies installation.
         """
-        assert self.DEPENDENCY_PUBLIC_ID in self.load_config().protocols
+        assert self.DEPENDENCY_PUBLIC_ID in {
+            p.without_hash() for p in self.load_config().protocols
```

Comment:
Ideally the old tests would pass so we maintain full compatibility. Can it be realised by having `with_hash` instead of `without_hash`?

Reply:
I this might help you understand: https://github.com/valory-xyz/open-aea/pull/103#discussion_r844255533